cardano-db-sync/src/Cardano/DbSync/DbAction.hs

```diff
@@ -48,7 +47,9 @@ newDbActionQueue :: IO DbActionQueue
 newDbActionQueue =
   -- Use an odd number here so that the db_tip_height metric increments by this odd number
   -- when syncing, instead of incrementing by say 100.
-  DbActionQueue <$> TBQ.newTBQueueIO 117
+  -- The pipeline queue in the LocalChainSync machinery is 50 elements long
+  -- so we should not exceed that.
+  DbActionQueue <$> TBQ.newTBQueueIO 47
```

Comment:
Why is it an issue to exceed the LocalChainSync queue?

Reply:
The the local chain sync protocol exceeds its depth, it causes a disconnect and the disconnect/reconnect cycle takes a non-zero amount of time. Before this change, with just a ledger state replay, I was seeing this for nearly every epoch.